opg-phpcs-1604/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.service.opg.digital/opg-php-fpm-1604
+
+RUN apt-get update && apt-get install -y \
```

Comment:
These would be test containers and not something that we would want to go into production. They're linting their code, let's give them that at least ;)